src/components/reportPage/batch-job.css

```diff
@@ -82,6 +82,59 @@
   border-color: #ccc transparent;
 }
 
+.loading-style {
+  position: fixed;
+  background: #00000050;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+}
+
+.exit-help-button {
+  position: absolute;
+  top: 10px;
+  right: 10px;
+}
+
+.help-button {
+  margin: auto;
+  width: 20%;
+  font-size: 1.05rem;
+  padding: 0.3rem;
+  border-radius: 2rem;
+  border: 0.02rem solid;
+  text-align: center;
+  font-weight: 600;
```

Comment:
Can you add `cursor: pointer;` here so that it looks like a button when you hover over it?

Reply:
yup done! :)